src/Uno.UI/Microsoft/UI/Xaml/Controls/PagerControl/PagerControl.cs

```diff
@@ -77,7 +77,7 @@ public PagerControl()
 			templateSettings.SetValue(PagerControlTemplateSettings.NumberPanelItemsProperty, m_numberPanelElements);
 			SetValue(TemplateSettingsProperty, templateSettings);
 
-			DefaultStyleKey = typeof(PagerControl);
+			SetDefaultStyleKey(this);
 		}
 
 		~PagerControl()
```

Comment:
I just noticed, @MartinZikmund this block below is likely to cause instability as the finalizer is run outside the dispatched and the dispose manipulates DependencyObjects.

Reply:
@jeromelaban This is an existing problem right? Not introduced by the changes in this PR. I think we can get it merged and open an issue to fix this separately?

Or probably @MartinZikmund can fix that in this PR if that's important. I don't know the correct fix :(

Reply:
Yes, we can create a different issue and fix it separately.

Reply:
@jeromelaban Great. I opened https://github.com/unoplatform/uno/issues/7933. I think the PR is ready for review.